change/@fluentui-react-charting-1d5fee55-3465-4900-848c-0885beef9400.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=25964&commitId=529d450742cea52bd0fafa6111682add5e4e1bce&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1774 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image Converged.Image Layout Fit.normal.chromium.png|50060| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>